### PR TITLE
[fuchsia] Add integration test for reproduction, and pass additional

### PR DESCRIPTION
**Pull request by flowerhack**
libfuzzer arguments.

This is some follow-up / housekeeping work to make sure reproduction &
libfuzzer arguments work well for Fuchsia!

### CODE REVIEW

File: src/python/tests/core/bot/fuzzers/libFuzzer/libfuzzer_launcher_integration_test.py [hunk 1]
```diff
@@ -710,14 +710,22 @@ def tearDown(self):
       'Temporarily disabling the Fuchsia test until build size reduced.')
```
Comment (oliverchang) [inline src/python/tests/core/bot/fuzzers/libFuzzer/libfuzzer_launcher_integration_test.py]: I can do this, but just a FYI that this test will be moving to engine_test.py

Comment (flowerhack) [reply src/python/tests/core/bot/fuzzers/libFuzzer/libfuzzer_launcher_integration_test.py]: good to know, thanks!

**Pull request merged**